locale/en_US/user.xml

```diff
@@ -71,19 +69,20 @@
 	<message key="user.logOutAs">Logout as</message>
 	<message key="user.hello">Hello, </message>
 	<message key="user.mailingAddress">Mailing Address</message>
-	<message key="user.middleName">Middle Name</message>
 	<message key="user.myAccount">My Account</message>
 	<message key="user.name">Name</message>
 	<message key="user.password">Password</message>
 	<message key="user.repeatPassword">Repeat password</message>
 	<message key="user.phone">Phone</message>
+	<message key="user.preferedPublicName">Prefered Public Name</message>
+	<message key="user.preferedPublicName.description">The prefered public name will be used ... Middle name, salutation and suffix can also be added here.</message>
 	<message key="user.profile.changePasswordInstructions">Enter your current and new passwords below to change the password for your account.</message>
 	<message key="user.profile.editProfile">Edit Profile</message>
 	<message key="user.profile.form.countryRequired">A country is required.</message>
 	<message key="user.profile.form.emailRequired">A valid email address is required.</message>
 	<message key="user.profile.form.affiliationRequired">An affiliation is required.</message>
-	<message key="user.profile.form.firstNameRequired">A first name is required.</message>
-	<message key="user.profile.form.lastNameRequired">A last name is required.</message>
+	<message key="user.profile.form.givenNameRequired">A first name is required.</message>
+	<message key="user.profile.form.givenNameRequired.locale">A first name is required for a locale in which only the last name exists.</message>
```

Comment:
> Please add a Given Name for the {$localeName} language.

Reply:
Here, i.e. for the form validators, we also cannot provide parameters/variables in the error messages :-\ Would you maybe have any other suggestion? THANKS!!!

Reply:
I think it's great! THANKS!!! :-)))

Reply:
I just took family instead of last name

Reply:
Just FYI -- adding parameters to `FormValidator` uninvasively wouldn't be hard. Add a `setParameters` function that stores an array as a member variable, then add it to the `getMessage` call as a parameter to `__`.

Reply:
I can do that now or later, as you wished... :-)
If no preference here, I will leave it as is for now...